ggshield/cmd/sca/scan/sca_scan_utils.py

```diff
@@ -67,7 +67,10 @@ def sca_scan_all(ctx: click.Context, directory: Path) -> SCAScanAllOutput:
         empty_output.status_code = sca_filter_status_code
         return empty_output
 
-    scan_parameters = SCAScanParameters(config.user_config.sca.minimum_severity)
+    scan_parameters = SCAScanParameters(
+        minimum_severity=config.user_config.sca.minimum_severity,
+        ignored_vulnerabilities=config.user_config.sca.ignored_vulnerabilities,
+    )
```

Comment:
Nit: could be added in a specific function that will be called here and in `sca_sca_diff` to ensure we get the same parametrization.